### PR TITLE
Cleaned up module and added support for profiles

### PR DESCRIPTION
**Pull request by tuxudo**
Contains new client script

### CODE REVIEW

File: scripts/uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Remove munkireportinfo script
+rm -f "${MUNKIPATH}preflight.d/munkireportinfo"
```
Comment (bochoven) [inline scripts/uninstall.sh:4]: I don't think this file ever existed :-D

Comment (tuxudo) [reply scripts/uninstall.sh:4]: I believe you're correct. Wouldn't we want to remove it if it exists in the future?

**Comment by bochoven** [comment]
LGTM did you test this? Maybe let some other people test.

**Comment by tuxudo** [comment]
I did, works exactly like the old module. @Ofir Gal tested it on some of his machines as well

**Comment by bochoven** [comment]
Could you add this change to the CHANGELOG in Munkireport-php?

**Comment by tuxudo** [comment]
This file via a PR? https://github.com/munkireport/munkireport-php/blob/master/CHANGELOG.md

**Pull request merged**